docs/usage/configuration-options.md

```diff
@@ -596,19 +596,12 @@ Check out our [Node.js documentation](https://docs.renovatebot.com/node) for a c
 
 The following dep types are currently supported by the npm manager :
 
-### dependencies
-
-### devDependencies
-
-### optionalDependencies
-
-### peerDependencies
-
-### engines
-
-### volta
-
-Renovate will update any node and yarn version specified under `volta`.
+1. `dependencies`
+2. `devDependencies`
+3. `optionalDependencies`
+4. `peerDependencies`
+5. `engines`
+6. `volta` : Renovate will update any node and yarn version specified under `volta`.
```

Comment:
Make this an unordered list. And for engines you can list what we cover there too (node, npm and yarn IIRC)